src/pages/Account/name/NameSetting.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import gql from 'graphql-tag'
+import { graphql } from 'react-apollo'
+import { store } from '../../../redux'
+import { showNotification } from '../../../actions/rootActions'
+import EditableInputSetting from '../EditableInputSetting'
+import styles from '../userName/UsernameSettings.module.scss'
+
+const CHANGE_NAME_MUTATION = gql`
+  mutation changeName($value: String!) {
+    changeName(name: $value) {
+      name
+    }
+  }
+`
+
+const validateName = name => {
+  if (name && name.length < 3) {
+    return 'Name should be at least 3 characters long'
+  }
```

Comment:
`[D-0-2]` If the `name` is empty string/falsy, function will return `undefined`. Is it desired behaviour?

Reply:
Yes names and usernames are nullable.